src/firefly/js/charts/ui/HistogramPlotly.jsx

```diff
@@ -156,7 +156,7 @@ export class HistogramPlotly extends Component {
             plotlyLayout: {
                 height,
                 width,
-                hovermode: 'x',
+                hovermode: 'compare',
```

Comment:
This file is not used and will be removed soon. FireflyHistogram is handling histogram creation.